package.json

```diff
@@ -21,10 +21,10 @@
   },
   "dependencies": {
     "@alaskaairux/icons": "^4.44.1",
-    "@aurodesignsystem/auro-dropdown": "^3.0.2",
+    "@aurodesignsystem/auro-dropdown": "^3.1.0",
     "@aurodesignsystem/auro-formvalidation": "^1.0.4",
     "@aurodesignsystem/auro-input": "^4.0.0",
-    "@aurodesignsystem/auro-library": "^2.8.0",
+    "@aurodesignsystem/auro-library": "^2.10.1",
```

Comment:
I might be a little worried updating to the latest auro-library. Might break things.

Reply:
Heard, I'll revert the library change for now. It's. not strictly necessary for this bug fix.